.github/workflows/maven.yml

```diff
@@ -23,19 +23,27 @@ jobs:
       - name: Checkout sources
         uses: actions/checkout@v1
 
-      - name: Build with Maven
+      - name: Build datasource client with Maven
         run: mvn --batch-mode -Pjacoco install
+        working-directory: ./case-datasource
 
-      - name: Run SonarCloud analysis
+      #We only run the sonar analysis on the server directory because we can't push 2 sonar result to the same sonar repository
+      #TODO add the analysis on the client when the client and server are split into 2 differents github repo
+      - name: Run SonarCloud analysis on case server
         run: >
           mvn --batch-mode -DskipTests sonar:sonar
           -Dsonar.host.url=https://sonarcloud.io
           -Dsonar.organization=powsybl-ci-github
           -Dsonar.projectKey=com.powsybl:powsybl-case-server
+        working-directory: ./case-server
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
 
+      - name: Build case server with Maven
```

Comment:
Is it needed to run the SonarCloud analysis after the step which build the case sserver ?

Reply:
no why ?